teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(specialization, age, name: 'Unknown')
```

Comment:
- [ ] Good job so far, :clap: , However, as requested by the previous reviewer and as part of the project requirement, Please ensure that you add the `parent_permission` and assign a default value to your constructure as in the example below: :slightly_smiling_face: 
`initialize(age, specialization, name = "Unknown", parent_permission = true)`

Reply:
Please take your job seriously. See the requirement:
- Create a class Teacher with the following:

1. Inherits from Person.
2. Constructor extends parent's constructor by adding @specialization and a parameter for it.
3. Override can_use_services? so it always returns true.

Now the parent_permission parameter was never used because it's always supposed to be true.